src/main/resources/com/google/api/codegen/go/main.snip

```diff
@@ -8,17 +8,9 @@
     package {@view.packageName}
 
     import (
-        "fmt"
-        "runtime"
-        "time"
-
-        "github.com/googleapis/gax-go"
-        "golang.org/x/net/context"
-        "google.golang.org/api/option"
-        "google.golang.org/api/transport"
-        "google.golang.org/grpc"
-        "google.golang.org/grpc/codes"
-        "google.golang.org/grpc/metadata"
+    @join imp : view.imports
+    {@imp}
```

Comment:
I'm not convinced that the blank line will be indented. If you have experimented and verified that it does indent the blank line when this is indented, then go ahead and keep it unindented.